website/docs/d/storage_account.html.markdown

```diff
@@ -35,13 +35,13 @@ output "storage_account_tier" {
 
 * `location` - The Azure location where the Storage Account exists
 
-* `account_kind` - Defines the Kind of account, either `BlobStorage` or `Storage`.
+* `account_kind` - The Kind of account.
 
-* `account_tier` - Defines the Tier of this storage account.
+* `account_tier` - The Tier of this storage account.
 
-* `account_replication_type` - Defines the type of replication used for this storage account.
+* `account_replication_type` - The type of replication used for this storage account.
 
-* `access_tier` - Defines the access tier for `BlobStorage` accounts.
```

Comment:
We should remove "Defines" in all fields' description, is it a documentation style? Do we need to update the other documents?

Reply:
As mentioned above this is a datasource, so it doesn't define the access tier, just returns what the access tier is.